.npmrc

```diff
@@ -1,18 +1,13 @@
 auto-install-peers=true
 shared-workspace-lockfile=true
 
-# @vue/component-compiler-utils
-hoist-pattern[]=sass
+# for md-editor-v3
 hoist-pattern[]=*@codemirror*
 hoist-pattern[]=*@lezer*
 
-# eslint (public so they are picked up by IDEs)
-public-hoist-pattern[]=*eslint*
-public-hoist-pattern[]=*prettier*
-
-# Make ODS available to tests, even if the main package does not require it
-public-hoist-pattern[]=@opencloud-eu/design-system
-
 # c.f. https://github.com/vuejs/test-utils/pull/2197
 hoist-pattern[]=@vue/server-renderer
 
+# eslint and prettier (public so they are picked up by IDEs)
```

Comment:
Aaaah! Ooooh! Thank you ❤️

Reply:
I just moved the code block and clarified the comment though 😅 So you probably still need the manual config in your IDE for whatever reason.